.github/workflows/pr.yml

```diff
@@ -0,0 +1,39 @@
+name: orka-go-ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+env:
+  GOLANG_VERSION: "1.21.6"
+
+jobs:
+  ci:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOLANG_VERSION }}
+
+      - name: Check for unused packages
+        run: |
+          make tidy
+          git diff --exit-code
+
+      - name: Format code
+        run: |
+          make fmt
+          git diff --exit-code
+
+      - name: Check for unused dependencies
```

Comment:
Looks like we have this step twice (already above on line 22)

Reply:
Whoops, thanks